examples/wearable/UIComponents/contents/navcontrols/indexscrollbar/indexScrollbar.js

```diff
@@ -46,17 +47,43 @@
 			});
 		}
 
+		listview = tau.engine.getBinding(listviewElement);
+
 		// Add IndexScrollbar index "select" event handler.
 		indexScrollbarElement.addEventListener("select", function (ev) {
 			var divider,
-				idx = ev.detail.index;
+				idx = ev.detail.index.trim(),
+				index;
 
 			divider = dividers[idx];
-			if (divider && scroller) {
-				// Scroll to the ui-listview-divider element
-				scroller.scrollTop = divider.offsetTop - scroller.offsetTop;
+
+			if (!isCircle) {
+				if (divider && scroller) {
+					// Scroll to the ui-listview-divider element
+					scroller.scrollTop = divider.offsetTop - scroller.offsetTop;
+				}
+			} else {
+				index = listview.findItemIndexByDivider(divider);
+				if (index > -1) {
+					listview.scrollToPosition(index);
+				}
+			}
+		});
+
+		indexScrollbarElement.addEventListener("show", function () {
```

Comment:
Are those "show" and "hide" events are needed to be handled to solve this issue? Wondering if they shouldn't be part of different commit/PR if possible.

Reply:
This is base for this change.